product-scenarios/pom.xml

```diff
@@ -74,16 +74,16 @@
 
     <modules>
         <module>scenarios-commons</module>
-        <module>1-integrating-systems-that-communicate-in-heterogeneous-message-formats</module>
```

Comment:
We are still testing scenario 10 test cases in test-grid environment. Please note that following modules have been commented as it is unnecessary to run commented modules to test scenario 10. Following modules will be uncommented before this pr is merged.